src/riak_kv.proto

```diff
@@ -389,14 +389,11 @@ enum TsColumnType {
   FLOAT = 2;
```

Comment:
Went with DOUBLE and SINT64 (reasoning that there might be sint32 or uint64 but simple float rather than float32).